src/halmos/cheatcodes.py

```diff
@@ -652,11 +652,13 @@ def handle(sevm, ex, arg: ByteVec, stack, step_id) -> ByteVec | None:
                 ex, store_account, stack, step_id, branching=False
             )
 
-            if store_account_alias is None:
-                error_msg = f"vm.store() is not allowed for a nonexistent account: {hexify(store_account)}"
-                raise HalmosException(error_msg)
+            if store_account_alias is not None:
+                store_account = store_account_alias
+
+            # vm.store() initializes storage if the account is not already initialized
+            ex.storage.setdefault(store_account, sevm.mk_storagedata())
 
-            sevm.sstore(ex, store_account_alias, store_slot, store_value)
+            sevm.sstore(ex, store_account, store_slot, store_value)
```

Comment:
allowing vm.store() on empty accounts needs further discussions. can we focus on the vm.etch() fix in this pr, without making the above changes to vm.store()?

Reply:
ok, reverted the vm.store change in [55fba55](https://github.com/a16z/halmos/pull/363/commits/55fba554caaae8b14e350b6ebde366479741643c)